plugin/pkg/admission/persistentvolume/label/admission_test.go

```diff
@@ -75,6 +77,14 @@ func mockVolumeLabels(labels map[string]string) *mockVolumes {
 	return &mockVolumes{volumeLabels: labels}
 }
 
+func mockCloudConfig() io.Reader {
+	gceConfigData := `
+[Global]
+multizone=true
+        `
+	return strings.NewReader(gceConfigData)
+}
+
```

Comment:
Ignore this for now. I added this in preparation for adding some tests in admission plugin as well.